examples/advanced/vertical_xgboost/jobs/vertical_xgboost_base/app/config/config_fed_server.json

```diff
@@ -0,0 +1,24 @@
+{
+  "format_version": 2,
+  "server": {
+    "heart_beat_timeout": 600
+  },
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "workflows": [
+    {
+      "id": "controller",
+      "name": "DhPSIController",
+      "args": {
+      }
+    },
+    {
+      "id": "xgb_controller",
+      "path": "nvflare.app_opt.xgboost.histogram_based.controller.XGBFedController",
+      "args": {
+        "train_timeout": 30000
+      }
+    }
+  ],
```

Comment:
If we want to have PSI run first then vertical XGBoost then this config is good.

BUT, will there be a case where users first run PSI to join the dataset.
And later keep running vertical XGBoost to get better accuracy?

@chesterxgchen @ZiyueXu77 can you comment on this thanks

Reply:
@holgerroth what do you think? Should we use 2 jobs or 1?

Reply:
Maybe we just use 2 jobs as in here: https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/vertical_federated_learning/cifar10-splitnn/jobs